lighthouse-core/gather/gather-runner.js

```diff
@@ -336,10 +348,10 @@ class GatherRunner {
       passContext.options = gathererDefn.options || {};
 
       // If there was a pageLoadError, fail every afterPass with it rather than bail completely.
-      const artifactPromise = pageLoadError ?
-        Promise.reject(pageLoadError) :
-        // Wrap gatherer response in promise, whether rejected or not.
-        Promise.resolve().then(_ => gatherer.afterPass(passContext, passData));
+      const artifactPromise = pageLoadError
+        ? Promise.reject(pageLoadError)
```

Comment:
did prettier get loose? This is bad (I'm surprised we don't lint it) and the concat thing above is definitely worse :)

Reply:
prettier! get back in your cage.